bpf/lib/maps.h

```diff
@@ -199,6 +199,30 @@ struct {
 	__uint(max_entries, 1);
 } ENCRYPT_MAP __section_maps_btf;
 
+struct node_key {
+	__u16 pad1;
+	__u8 pad2;
+	__u8 family;
+	union {
```

Comment:
Agree. I've created https://github.com/cilium/cilium/issues/23657 for this, as a good-first-issue. Hopefully someone will pick it up, otherwise I can quickly do it at some point.